contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -135,107 +337,345 @@ NXmpes(NXobject):
+        sample_preparation(NXphysical_process):
+          exists: recommended
+          doc: |
+            Details about the sample preparation for the MPES experiment (e.g. UHV cleaving,
+            in-situ growth, sputtering/annealing, etc.).
+          start_time(NX_DATE_TIME):
+          end_time(NX_DATE_TIME):
+            exists: recommended
+          method:
+            exists: recommended
+            doc: |
+              Details about the method of sample preparation before the MPES experiment.
+      temperature(NXenvironment):
+        exists: recommended
         doc: |
-          Description of the surface preparation technique for the XPS experiment, i.e.
-          UHV cleaving, in-situ growth, sputtering/annealing etc. Ideally, a full report
-          of the previous operations, in any format(NXnote allows to add pictures, audio,
-          movies). Alternatively, a reference to the location or a unique identifier or
-          other metadata file. In the case these are not available, free-text description.
-      temperature(NX_FLOAT):
-        unit: NX_TEMPERATURE
+          Sample temperature (either controlled or just measured).
+        temperature_sensor(NXsensor):
+          doc: |
+            Temperature sensor measuring the sample temperature.
+            This should be a link to /entry/instrument/manipulator/temperature_sensor.
+        sample_heater(NXactuator):
+          exists: optional
+          doc: |
+            Device to heat the sample.
+            This should be a link to /entry/instrument/manipulator/sample_heater.
+      gas_pressure(NXenvironment):
+        exists: recommended
         doc: |
-          In the case of a fixed temperature measurement this is the scalar temperature of
-          the sample. In the case of an experiment in which the temperature is changed and
-          recoded, this is an array of length m of temperatures. This should be a link to
-          /entry/instrument/manipulator/sample_temperature.
-      situation:
-        enumeration: [vacuum, inert atmosphere, oxidising atmosphere, reducing atmosphere]
-      gas_pressure(NX_FLOAT):
-        unit: NX_PRESSURE
-      bias(NX_FLOAT):
-        unit: NX_VOLTAGE
+          Gas pressure surrounding the sample.
+        pressure_gauge(NXsensor):
+          doc: |
+            Gauge measuring the gas pressure.
+            
+            This should be a link to /entry/instrument/pressure_gauge.
+      bias(NXenvironment):
+        exists: recommended
+        doc:
+        - |
+          Bias of the sample with respect to analyser ground.
+        - |
+          xref:
+            spec: ISO 18115-1:2023
+            term: 8.41
+            url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:8.41
+        voltmeter(NXsensor):
+          doc: |
+            Sensor setting/measuring the applied voltage.
+            
+            This should be a link to /entry/instrument/manipulator/sample_bias_voltmeter.
```

Comment:
maybe rather the potentiostat?

Reply:
The idea was to have both the measurement and the actutation.

I have added a sample_bias_potentiostat here and in `NXmpes/NXmanipulator`.